tox.ini

```
@@ -47,6 +47,7 @@ commands =
 description = Enforce quality standards under {basepython} ({envpython})
 install_command = pip install {opts} {packages}
 commands =
+  pre-commit run -a {posargs}
```

I think in the past you asked me not to use args like `-a` with unclear meanings in scripts. Now, I'd like you to do the same.
Also, I would expect this to make multiple invocations and not to bring a feature of customizing only one linter invocation, so I'd drop `{posargs}` too:
```suggestion
  {envpython} -m \
    pre_commit run \
    markdownlint \
    --show-diff-on-failure \
    --all-files -v
  {envpython} -m \
    pre_commit run \
    codespell \
    --show-diff-on-failure \
    --all-files -v
  {envpython} -m \
    pre_commit run \
    flake8 \
    --show-diff-on-failure \
    --all-files -v
  {envpython} -m \
    pre_commit run \
    yamllint \
    --show-diff-on-failure \
    --all-files -v
  {envpython} -m \
    pre_commit run \
    mypy \
    --show-diff-on-failure \
    --all-files -v
```